pkg/porter/create.go

```diff
@@ -1,43 +1,73 @@
 package porter
 
 import (
+	"errors"
 	"fmt"
 	"os"
 	"path/filepath"
+	"strings"
 
 	"get.porter.sh/porter/pkg"
 	"get.porter.sh/porter/pkg/config"
 )
 
-func (p *Porter) Create() error {
-	fmt.Fprintln(p.Out, "creating porter configuration in the current directory")
+// Create creates a new bundle configuration with the specified bundleName. A directory with the given bundleName will be created if it does not already exist.
+// If bundleName is the empty string, the configuration will be created in the current directory, and the name will be "porter-hello".
+func (p *Porter) Create(bundleName string) error {
+	// Normalize the bundleName by removing trailing slashes
+	bundleName = strings.TrimSuffix(bundleName, "/")
 
-	err := p.CopyTemplate(p.Templates.GetManifest, config.Name)
+	// If given a bundleName, create directory if it doesn't exist
+	if bundleName != "" {
+		_, err := os.Stat(bundleName)
+		if err != nil && errors.Is(err, os.ErrNotExist) {
+			err = os.Mkdir(bundleName, os.ModePerm)
+			if err != nil {
```

Comment:
Your error is not nil so you don’t have to check it again. Your logic above can be changed to check if it is nil and the. Check if it is the error you are looking for. If it is still nil, you can fall through to the not nil case and not re check the error.

Reply:
This applies to line 23

Reply:
something like this? 	
```_, err := os.Stat(dir)
	if err != nil {
		if errors.Is(err, os.ErrNotExist) {
			err = os.MkdirAll(dir, os.ModePerm)
		}
		if err != nil {
			return fmt.Errorf("failed to create directory for bundle: %w", err)
		}
	}
```